backend/strapi-sync/sync.js

```diff
@@ -258,7 +258,13 @@ export async function createOrUpdateFeature(item) {
   let dbItem = await getItemByAttributes(Feature, { strapiId: item.id });
 
   if (dbItem) {
-    dbItem.name = item.attributes.parkSubArea;
+    // if dbItems has campgroundId, don't update the name
+    // there is a script to create campgrounds and assign features to them
+    // this script will rename the feature to fit within the campground
+    if (!dbItem.campgroundId) {
+      dbItem.name = item.attributes.parkSubArea;
+    }
```

Comment:
I noticed while pulling in the Section/Mgmt Area details for the CSV, we need to make sure we're running `dbItem.save()` after making changes, don't we?

Reply:
Yes, you're right. I could have sworn I fixed that somewhere. But, anyway just pushed the fix.